### PR TITLE
Update robots.cfg

### PR DESCRIPTION
**Pull request by perusiongreg**
Remove MSIE, GECKO from NotRobotUA as many bots use this UA, also added some new bot names

### CODE REVIEW

**Review by hexfusion — APPROVED** [review]
Merry Christmas and Happy New Year :)

File: dist/robots.cfg [hunk 1]
```diff
@@ -108,6 +112,7 @@ RobotUA <<EOR
     panscient.com,
     ParaSite,
     pavuk,
+	Pinterestbot,
```
Comment (jonjensen) [inline dist/robots.cfg:115]: @perusiongreg Looks like there's different indentation for Pinterestbot. Can you make it indent like the surrounding lines?

**Review by jonjensen — CHANGES_REQUESTED** [review]
Overall looks good to me.

I do wonder about the Microsoft Outlook new bot designation. If someone receives an email in Outlook with a link to a sale at an Interchange ecommerce site and clicks on it, do they stay in the Outlook internal browser and get denied a session? Or does it open in a normal browser such as Edge or Chrome?

I'm going to guess it opens in a real browser, but it'd be nice to know for sure.

File: dist/robots.cfg (hunk 1 above)
Comment (perusiongreg) [reply dist/robots.cfg]: fixed that. was tab vs spaces

Comment (perusiongreg) [reply dist/robots.cfg]: Regarding the Outlook, I think it pertains more to the Preview functions that query the site when an email is viewed within the application .. that preview uses the Outlook UA, same as BingPreview. I don't have Outlook to test against, so cannot say for sure, but this is on BS which is a high traffic sight and very "picky" and outspoken customers.  So at least we have good odds in our favor.

**Review by jonjensen — APPROVED** [review]

**Comment by jonjensen** [comment]
@perusiongreg Thanks!

Since we've got 2 approvals now, I'm going to merge this.

**Pull request merged**